mne/source_space.py

```diff
@@ -1136,12 +1136,8 @@ def vertex_to_mni(vertices, hemis, subject, subjects_dir=None, mode=None,
 
 
 @verbose
-def _read_talxfm(subject, subjects_dir, mode=None, verbose=None):
-    """Read MNI transform from FreeSurfer talairach.xfm file.
-
-    Adapted from freesurfer m-files. Altered to deal with Norig
-    and Torig correctly.
-    """
+def _read_talairach(subject, subjects_dir, mode=None, verbose=None):
```

Comment:
This does not use `mode` other than checking valid, that should be moved (back) to `_read_talxfm`